src/components/ratings_reviews/Ratings.jsx

```diff
@@ -18,43 +20,40 @@ function Ratings() {
       Average Ratings
       <h1>
         4.2
-        <div
-          className="stars"
-          style={{ '--rating': (Math.round(4.2 * 4) / 4).toFixed(2) }}
-        />
+        <Stars rating={(Math.round(4.2 * 4) / 4).toFixed(2)} />
```

Comment:
As discussed, later down the road let's move this formula inside of the Stars component. My section also calls for rounding to nearest .25 so this would be super helpful